### PR TITLE
Add scroll to top button on search page

### LINKED ISSUE #7: Add a scroll to top shortcut !
image

Currently , there is no scroll to top shortcut . Add it on the above page

### PR DESCRIPTION
**Pull request by kavin25**
fixes #7

### CODE REVIEW

File: pages/search.tsx [hunk 1]
```diff
@@ -145,6 +161,14 @@ export default function search({ image_list, error }: search_props) {
             {/* starts here */}
           </div>
         </div>
+        {showScrollButton && (
+          <button
+            className="fixed bottom-10 right-10 w-16 h-16 rounded-full bg-black text-white"
+            onClick={scrollToTop}
+          >
+            &#8679;
```
Comment (parthpanchal123) [inline pages/search.tsx:169]: It would be great if you can add a custom icon. The current symbol looks a bit off .

**Review by parthpanchal123 — COMMENTED** [review]
It would be great if you can add a custom icon. The current symbol looks a bit off .

File: pages/search.tsx (hunk 1 above)
Comment (kavin25) [reply pages/search.tsx:169]: Sure, let me see

**Comment by kavin25** [comment]
@parthpanchal123 made the changes

**Comment by parthpanchal123** [comment]
image

@kavin25 Just update crossorigin to crossOrigin and referrerpolicy to referrerPolicy  on search.tsx  , line no 120 and 121 commit

**Review by parthpanchal123 — APPROVED** [review]

**Comment by kavin25** [comment]
@parthpanchal123 Looks like someone already has?

**Comment by parthpanchal123** [comment]
@kavin25 Thank you for your awesome contribution :partying_face:  . Highly appreciate your efforts :fire: :100:

**Comment by parthpanchal123** [comment]
It'll be great if you can :star: the repo if you liked it .

**Pull request merged**